resources/views/admin/user/edit.blade.php

```diff
@@ -0,0 +1,54 @@
+<?php ?>
+@extends('admin.layout.index')
+
+@section('content')
+    <div id="page-wrapper">
```

Comment:
If edit and create are almost the same , try to use only 1 blade file

Reply:
No, we have not filled in the frame yet, so the message box and information that will appear in the edit screen text box will be different from the create screen.

Reply:
Too much duplicate code , try to reduce , use only 1 file , depend on usage on controller , display different content